recommendations/index.md

```diff
@@ -149,3 +149,53 @@ Although JSON API does not specify the format of date and time fields, it is
 recommended that servers align with ISO 8601. [This W3C
 NOTE](http://www.w3.org/TR/NOTE-datetime) provides an overview of the
 recommended formats.
+=======
+
+## Asynchronous Processing <a href="#asynchronous-processing" id="asynchronous-processing" class="headerlink"></a>
+
+Consider a situation when you need to create a resource and the operation takes long time to complete.
+
+The request **SHOULD** return a status `202 Accepted` with a link in Header/Location.
+
+```text
+HTTP/1.1 202 Accepted
+Location: https://example.com/photos/queue/5234
```

Comment:
Fwiw, HTTP [does not define](https://tools.ietf.org/html/rfc7231#section-7.1.2) the meaning of the `Location` header for a `202` response, so it's not clear that this is appropriate here. (It only defines it for 201 and 3xx.) A more appropriate header might be `Link` with some relation. I'll need to think about this. 

//cc @steveklabnik